specification/reservations/resource-manager/Microsoft.Capacity/preview/2018-06-01/reservations.json

```diff
@@ -780,7 +740,9 @@
             "x-ms-azure-resource": true,
             "properties": {
                 "location": {
-                    "$ref": "#/definitions/Location"
+                    "type": "string",
+                    "readOnly": true,
```

Comment:
Just FYI, this is non breaking only because it's a return type. If it was an input type, adding "readOnly" is dangerous.